chain-impl-mockchain/src/testing/verifiers/ledger_verifier.rs

```diff
@@ -189,6 +192,26 @@ impl LedgerStateVerifier {
     pub fn votes(&self) -> VotesVerifier {
         VotesVerifier::new(self.ledger.active_vote_plans())
     }
+
+    #[cfg(feature = "evm")]
+    pub fn is_mapped_to_evm(&self, evm_mapping: &EvmMapping) -> &Self {
```

Comment:
Nitpick:

It would be great if evm related verifications would be done on child level of LedgerVerifier, as this struct was mainly design to just be a container of child verifiers. In essence i would like to call:

```
  let ledger_verifier = ...;
  ledger_verifier.evm().is_mapped(bob_mapping)
```

and LedgerVerifier should have sub verifier for emv only:

```
 impl LedgerVerifier {
  
 ...
 
 
 pub fn evm(self) -> EvmVerifier {
    EvmVerifier::new(...)
 }
```
So we will be prepared for adding more evm related verifications.

Reply:
Also we will can remove one `#[cfg(feature = "evm")]` for free!